<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZTestOptionDialog</class>
 <widget class="QDialog" name="ZTestOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Test Choices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="normalTestRadioButton">
        <property name="text">
         <string>Normal Test</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="unitTestRadioButton">
        <property name="text">
         <string>Unit Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="commandTestRadioButton">
        <property name="text">
         <string>Command line test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="stressTestRadioButton">
        <property name="text">
         <string>Stress Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="crashTestRadioButton">
        <property name="text">
         <string>Crash Test</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>stressTestRadioButton</zorder>
     <zorder>normalTestRadioButton</zorder>
     <zorder>unitTestRadioButton</zorder>
     <zorder>crashTestRadioButton</zorder>
     <zorder>commandTestRadioButton</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZTestOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZTestOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
